en/docs/guides/set-up-a-governed-multi-model-llm-proxy-with-cost-controls-and-failover.md

```
@@ -0,0 +1,402 @@
+    "model": "<AZURE-MODEL-NAME>",
+    "input": "What is the capital of France?"
+  }'
+```
+
+<video controls width="100%">
+  <source src="../../assets/clips/run_claude.mp4" type="video/mp4">
+  Your browser does not support the video tag.
+</video>
+
+
+**Expected response: ** HTTP `200` with a response object. The answer appears in `output[0].content[0].text`. The proxy distributes the request to the next model in the rotation.
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)